Ginger/GingerCoreNET/Run/RunSetActions/RunSetActionHTMLReportSendEmailOperations.cs

```diff
@@ -90,7 +90,15 @@ public void Execute(IReportInfo RI)
                 {
                     //Reporter.ToLog(eLogLevel.DEBUG, "Run set operation send Email: Using LiteDB and using new WebReportGenerator");
                     reportsResultFolder = Path.Combine(ExtensionMethods.GetReportDirectory(currentConf.HTMLReportsFolder), "Reports");
-                    reportsResultFolder = Path.Combine(reportsResultFolder, $"{General.RemoveInvalidFileNameChars(WorkSpace.Instance.RunsetExecutor.RunSetConfig.Name)}_{DateTime.UtcNow.ToString("yyyymmddhhmmssfff")}");
+                    string reportTempPath = Path.Combine(reportsResultFolder, $"{General.RemoveInvalidFileNameChars(WorkSpace.Instance.RunsetExecutor.RunSetConfig.Name)}_{DateTime.UtcNow.ToString("yyyymmddhhmmssfff")}");
+                    if (reportTempPath.Length >255)
+                    {
+                        reportsResultFolder = Path.Combine(reportsResultFolder, $"{General.RemoveInvalidFileNameChars(WorkSpace.Instance.RunsetExecutor.RunSetConfig.Name.Substring(0,5))}_{DateTime.UtcNow.ToString("yyyymmddhhmmssfff")}");
```

Comment:
make it smarter, calc (255- reportsResultFolder.length + {DateTime.UtcNow.ToString("yyyymmddhhmmssfff")}.length) and then figure out if remaninng chars number is smaller than run set name so take only part you can instead taking only 5 chars as default.